conformance/base/manifests.yaml

```diff
@@ -234,3 +241,59 @@ spec:
           env:
           - name: SERVE_SCTP_PORT_9005
             value: "foo"
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: centaur
+  namespace: network-policy-conformance-forbidden-forrest
+spec:
+  selector:
+    matchLabels:
+      conformance-house: forbidden-forrest
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        conformance-house: forbidden-forrest
+    spec:
+      hostNetwork: true
```

Comment:
this is the only daemon-set which is host-networked=true that helps us test nodes as peers..

Reply:
That should be fine, definitely the best way to test traffic to/from the node